packages/client/src/object/aggregateOrThrow.ts

```diff
@@ -85,8 +85,8 @@ export async function aggregateOrThrow<
   const ret: AggregationResultsWithGroups<Q, AO["select"], any> = result.data
     .map((entry) => {
       return {
-        group: entry.group as any,
-        values: legacyToModernSingleAggregationResult(entry),
+        $group: entry.group as any,
+        ...legacyToModernSingleAggregationResult(entry),
       };
     }) as any; // fixme
```

Comment:
Is there a good way to fix this, besides doing something like `as AggregationResultsWithGroups<Q, AO["select"], any>`?

Reply:
Ok I won't block merge on it, will think about it as I do other agg stuff potentially